system/system_info.go

```diff
@@ -22,7 +22,8 @@ func NewSystemInfo() *SystemInfo {
 	var cmd *exec.Cmd
 	switch runtime.GOOS {
 	case "linux":
-		cmd = exec.Command("uname -a", arg...)
+		arg = append(arg, "-a")
+		cmd = exec.Command("lsb_release", arg...)
```

Comment:
For my own edification as a non-Linux user, could you explain what the different here is?

Reply:
```sh
$uname -a
Linux Quantum 4.16.13-1-ARCH #1 SMP PREEMPT Thu May 31 23:29:29 UTC 2018 x86_64 GNU/Linux
```
Information from `uname` is not enough. There is nothing about the distro, such as Ubuntu, Fedora, etc. It just shows `GNU/Linux`.

But `lsb_realease -a` outputs details of a distro.
```sh
$lsb_release -a
LSB Version:	1.4
Distributor ID:	Arch
Description:	Arch Linux
Release:	rolling
Codename:	n/a
```

Reply:
Great, thanks.

Reply:
don't agree on that

Reply:
@deltaxflux Can you elaborate? I'm not a Linux guy, this is outside my domain.

Reply:
@deltaxflux Did you mean `lsb_release` not installed on some systems? 
How about  reading ` /etc/lsb-release`
But I don't know if /etc/lsb-release exists without lsb_release installed.

Reply:
I mean that lsb_release is not a core package of linux therefore we should use uname because it is shipped with Linux. I don't know if /etc/lsb-release exists if it is not installed but it doesn't make any sense, do you think so?

Reply:
uname is a core package but we cant get necessary info from it. 

Most info from uname can be got with std lib of golang, such as runtime.GOOS, runtime.GOARCH, os.Hostname()

I think its worth to have a try on lsb_release, and `screenfetch` is using lsb_release too

Reply:
Yeah agree